gallery/examples.js

```diff
@@ -252,6 +253,81 @@ var EXAMPLES = [
       },
       'data': {'url': 'data/cars.json'}
     }
+  },{
```

Comment:
I have similar concerns. We will see after next week.

On Fri, Oct 23, 2015 at 1:58 PM, Kanit Wongsuphasawat <
notifications@github.com> wrote:

> In gallery/examples.js
> <https://github.com/vega/vega-lite/pull/693#discussion_r42913620>:
>
> > @@ -252,6 +253,81 @@ var EXAMPLES = [
> >        },
> >        'data': {'url': 'data/cars.json'}
> >      }
> > +  },{
>
> I think refering for exemplar ones in vega-editor make sense. For ones
> that are more visual test (like histogram with point, line -- we should
> make sure that it is easy to create ones)
>
> Having it here, reduce the need to name the file nicely (and burden of
> renaming if the original name sucks, etc.) That said, this examples.js can
> become too damn long so I'm not against splitting it. Just wanna express
> potential downside of that here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/vega-lite/pull/693/files#r42913620>.
>